### DISCUSSION TITLE
Preparation soil data

### DISCUSSION THREAD

**Discussion by Mohammedhhu** [mhm-ufz/mHM#94] [Q&A]
Hello..

In per-proc, there are many files that need to be run according to the readme file.

So, I did not get the point of running those files.

**Comment by pallavhydro** [comment]
Hi @Mohammedhhu 

I could not understand your question. Could please you frame it clearly?

The README.md file in the mhm directory doesn't include anything about `pre-proc`.

Based on the discussion topic, it seems you had some comment/ question on soil data (??).

Pallav

**Comment by Mohammedhhu** [comment]
I mean, the step of preparing the soil file is not clear to me. 

As mentioned, in the README.txt file inside the SoilGrid folder. Some files need to run, so my question is, where can run those files?

![Capture](https://github.com/mhm-ufz/mHM/assets/77581066/a78cd555-99e7-4795-9264-62111317bad1)
